src/providers/python.rs

```diff
@@ -207,12 +226,14 @@ impl PythonProvider {
 
     fn is_django(app: &App, _env: &Environment) -> Result<bool> {
         let has_manage = app.includes_file("manage.py");
-        let imports_django = vec!["requirements.txt", "pyproject.toml"].iter().any(|f| {
-            app.read_file(f)
-                .unwrap_or_default()
-                .to_lowercase()
-                .contains("django")
-        });
+        let imports_django = vec!["requirements.txt", "pyproject.toml", "Pipfile"]
```

Comment:
I think this should use `uses_dep` instead (or the implementation of `uses_dep` should change to match this)